wgsl/index.bs

```diff
@@ -306,6 +306,32 @@ The WebGPU specification describes the consequences of each kind of error.
 
 TODO: Update the WebGPU spec, referring back to the three kinds of errors defined here.
 
+Shader module creation or pipeline creation will fail for a very complex [SHORTNAME] program:
+* A [=limit error=] occurs when a [[#limits|specified limit]] is exceeded.
+* An <dfn noexport>unexpected error</dfn> may occur
+    even when all [SHORTNAME] and WebGPU requirements have been satisfied.
```

Comment:
This is in conflict with the start of the section, where it says a program error is a failure to satisfy the requirements of this specification.

Reply:
Fixed in next commit.